sh/scripts/view_faucet_account.py

```diff
@@ -21,22 +22,28 @@
     type=args_validator.validate_network,
     )
 
+# CLI argument: node index.
+ARGS.add_argument(
```

Comment:
Is this new arg/flag being used in the script below? It doesn't seem like it

Reply:
Never mind! I saw that `get_network_node` is passed the args namespace, and it's used in there. All good!